pipeline/test_transform.py

```diff
@@ -1,4 +1,136 @@
-# pylint: skip-file
 """Test file for the transform pipeline."""
 
+# pylint: skip-file
 import pytest
+from unittest.mock import patch
+import pandas as pd
+
+from transform import (get_required_columns,
+                       check_valid_sale,
+                       clean_urls,
+                       handle_missing_values,
+                       validate_foreign_keys,
+                       get_id_mapping,
+                       filter_by_currency,
+                       add_foreign_key_column,
+                       clean_dataframe,
+                       export_dataframe)
+
+pytest.skip(allow_module_level=True)
+
+
+def test_get_required_columns_returns_the_required_columns(sample_df):
```

Comment:
Test name is a little bit of a mouthful, maybe something like:
```suggestion
def test_get_required_columns_returns_correctly(sample_df):
```